package/utils/busybox/Config-defaults.in

```diff
@@ -2240,7 +2249,6 @@ config BUSYBOX_DEFAULT_FEATURE_WATCHDOG_OPEN_TWICE
 config BUSYBOX_DEFAULT_FEATURE_IPV6
 	bool
 	default y if IPV6
-	default n
```

Comment:
You still need to leave the fallback default "no" there... (You are now removing it)

Reply:
Yeah, stupid oversight on my part. Fixed now, thanks!